wagon-providers/wagon-http-shared/src/main/java/org/apache/maven/wagon/shared/http/AbstractHttpClientWagon.java

```diff
@@ -279,6 +332,20 @@ public boolean isStreaming()
     private static final boolean SSL_ALLOW_ALL =
         Boolean.valueOf( System.getProperty( "maven.wagon.http.ssl.allowall", "false" ) );
 
+    /**
+     * If enabled, then the content-type HTTP header will be set using the file extension
+     * or the stream header to determine the type, <b>disabled by default</b>
```

Comment:
I'm not sure - is it disabled when property is set to `true` (which is set as default for `getProperty` call)?

Reply:
Nice catch, fixed the comment. property is TRUE by default (i.e. content-type will be determined and set)